swarm/pss/pss.go

```diff
@@ -25,7 +25,7 @@ import (
 // TODO: proper padding generation for messages
 const (
 	defaultPaddingByteSize     = 16
-	defaultMsgTTL              = time.Second * 8
+	defaultMsgTTL              = time.Second * 20
```

Comment:
@nolash not sure why this is set to 8 seconds, and whether we want to increase it. Based on my understanding of PSS, if the message is not delivered within 8 seconds, it will be discarded. If we run tests on one machine with high number of messages and nodes, I was hitting the 8 second limit.

Reply:
It's just a random setting. It was to be gauged by network tests and real life scenarios to find an appropriate value. In any case, it can be set in `PssParams`

Reply:
Thanks, I missed that it is defines from within `newServices()`. Reverted.